projects/angular/src/utils/_mixins.scss

```diff
@@ -24,13 +24,13 @@ $clr-outline-color-inverse: hsla(0, 0%, 100%, 0.09) !default;
 $clr-outline-blur: $clr_baselineRem_2px;
 $clr-outline-spread: $clr_baselineRem_2px;
 
-@mixin include-outline-style-form-fields(
-  $outline-color: $clr-outline-color,
-  $outline-blur: $clr-outline-blur,
-  $outline-spread: $clr-outline-spread
-) {
-  outline: 0;
-  box-shadow: 0 0 $outline-blur $outline-spread $outline-color;
+@mixin include-outline-style-form-fields() {
+  outline: -webkit-focus-ring-color solid $clr_baselineRem_2px;
```

Comment:
I assume this will work in Chrome and Edge, was it tested in Firefox and Safari?

Reply:
You are right. Though it works fine in Safari, Firefox needs "Highlight" color instead. And some more overrides, as we have some way too specific code in the normalize.css that needs to be overridden.
Changed! Thanks